tests/system_tests/test_launch/test_launch_sweep.py

```diff
@@ -9,11 +9,12 @@
 from wandb.sdk.launch.utils import LAUNCH_DEFAULT_PROJECT, construct_launch_spec
 
 
-@pytest.mark.parametrize(
-    "resource",
-    ["local-process", None],
-)
-def test_sweeps_on_launch(relay_server, user, resource, monkeypatch):
+def test_sweeps_on_launch(
```

Comment:
not sure about the sweeps on launch tests, assuming it is ok to remove the parameters

Reply:
this test was broken before and only passing due to a bug in the `relay_server` fixture, so should be fine to remove. It looks like a deprecated functionality that someone forgot to remove the test for.